cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -624,50 +678,78 @@ func TestDrain(t *testing.T) {
 		},
 	}
 
+	// run all tests for skipNodesWithCustomControllerPods=false
+	// vadasambar: TODO: remove this when we get rid of skipNodesWithCustomControllerPods
```

Comment:
nit: Let's format the TODO as `// TODO(vadasambar)` to be consistent with the rest of the codebase. As far as I can tell, we don't use the `// <github handle>: TODO:` pattern anywhere.

Reply:
Makes sense. I will update this.

Reply:
Updated.